securedrop/tests/test_2fa.py

```diff
@@ -28,28 +28,27 @@ def setUp(self):
     def tearDown(self):
         utils.env.teardown()
 
-    def _login_admin(self):
-        valid_token = self.admin.totp.now()
+    def _login_admin(self, token):
```

Comment:
Suggestion: since logging in with a valid token is such a common use case, it might be more ergonomic for developers writing tests to make `token` optional, e.g. `token=None`. If a token is passed, use it; otherwise, auto-generate a valid one.

Just a suggestion; I also think since there are relatively few tests in this file, it might be an unnecessary improvement/optimization. I'll approve this for merge either way, but encourage you to consider implementing this proposal--should be an easy change to make.